source/blog/2018-02-06-fedora-atomic-workstation.html.md

```diff
@@ -0,0 +1,172 @@
+                BaseCommit: a052d7482a186f1979f8bba90cfe1a1d0c13e75a43a416b580d2f2a83c18fe5a
+              GPGSignature: Valid signature by 860E19B0AFA800A1751881A6F55E7430F5282EE4
+           LayeredPackages: libvirt-client vim-enhanced
+```
+
+In contrast to `dnf/yum`-managed systems, which provide a &quot;bag of
+packages&quot; approach, `rpm-ostree status` allows to know *exactly* what we
+changed from the base OSTree commit, and how to go back to the initial state (in
+this case, `rpm-ostree uninstall libvirt-client vim-enhanced`).
+
+For completeness, I will mention that RPM-OSTree not only supports pure
+additions, but also *replacing* and *removing* base packages. All these changes
+can equally be tracked in `rpm-ostree status` and easily reverted.
+
+Additionally, because in the OSTree model configuration defaults are stored in
+`/usr/etc`, one can also perform a simple diff to figure out which defaults were
+changed (e.g. `diff /usr/etc /etc | less`).
+
+## 4. Upstream Testing
+
+Having updates delivered as concrete units allows the content provider to more
+easily test them, which translates into a more stable operating system for end
+users. In contrast, it's effectively impossible in the &quot;bag of
+packages&quot; approach to sanely test all permutations. This mostly means that
+testing is limited to the package-level.
+
+In an OSTree model however, it's possible to gate updates on higher-level
+integration tests such as what we've been doing in the two-week releases of
+Fedora Atomic Host. Note though that Atomic WS updates are not currently being
+gated on tests; we are still at the onset of these discussions.
+
+## 5. Automatic Updates
+
+In RPM-OSTree, we are currently working on adding an
+[automatic update feature](https://github.com/projectatomic/rpm-ostree/issues/247).
+Similar functionality on `dnf`-managed systems does
+[exist](https://fedoraproject.org/wiki/AutoUpdates). The difference is that the
+OSTree model allows us to eliminate or minimize
+[many of the issues](https://fedoraproject.org/wiki/AutoUpdates#Reasons_AGAINST_using_automatic_updates)
+involved in such a scheme. For example, the ability to roll back updates is
+critical here, and this is something that RPM-OSTree does with ease.
+
+In the future, we'd like to get to a point where (if enabled) the system
+automatically prepares updates in the background and "upgrading" simply involves
+rebooting your computer for minimal downtime.
+
+# How can I get started with Atomic WS?
+
+You can either
+[download the ISO](https://dl.fedoraproject.org/pub/fedora/linux/releases/27/WorkstationOstree/x86_64/iso/)
+and re-install from scratch, or you can convert your existing `dnf`-based system
+using the steps from
+[this](https://pagure.io/workstation-ostree-config/blob/master/f/README-install-inside.md)
+document. The latter also allows you to switch back and forth between the two
+models if you're not yet ready to dive in.
```

Comment:
should we mention/tell them they should upgrade after they've done the install from the ISO?